integration-commons/src/test/resources/application.yml

```diff
@@ -0,0 +1,183 @@
+
```

Comment:
This is used to test if the IntegrationProperties class is being correctly filled by String. I don't know if the test is useful, but the file is being used